bluetooth/src/commonTest/kotlin/BluetoothRequestMtuTest.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.bluetooth
+
+import com.splendo.kaluga.bluetooth.device.DeviceState
+import kotlinx.coroutines.flow.filterIsInstance
+import kotlinx.coroutines.flow.first
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class BluetoothRequestMtuTestTest : BluetoothFlowTest<Int>() {
+
+    override val flow = suspend {
+        setup(Setup.DEVICE)
+        bluetooth.devices()[device.identifier].mtu()
+    }
+
+    @Test
+    fun testRequestMtu() = testWithFlow {
+        val newMtu = 512
+
+        assertEquals(-1, connectionManager.mtu)
+
+        scanDevice()
+        bluetooth.startScanning()
+        action {
+            connectDevice(device)
+            assertTrue(bluetooth.devices()[device.identifier].requestMtu(newMtu))
```

Comment:
What actually `requestMtu` return value means? Looks like it is `isConnected`  status, but it is not obvious from the function signature and not obvious in tests what here `assertTrue`.  could you at least add a description in this (and other similar) assertion. Something like "Device should be connected" or "It expects device to be connected" (If my assumption about the meaning of return value is correct, of course).

Reply:
This method return true on success request otherwise false, I'll update doc than

Reply:
Added docs, and removed that confusing assertion (actually pointless, because it goes through successful mock anyway)